cmd/juju/model/destroy.go

```diff
@@ -433,7 +433,7 @@ func getModelStatus(ctx *cmd.Context, api DestroyModelAPI, tag names.ModelTag) (
 		err = status[0].Error
 	}
 	if err != nil {
-		if params.IsCodeNotFound(err) {
+		if errors.Is(err, errors.NotFound) {
```

Comment:
Is the error possibly a `modelerrors.NotFound`?

Reply:
doesn't seem to be